### PR TITLE
SJSU-1001, Finish Register & Login, Deploying to heroku

### PR DESCRIPTION
**Pull request by minhphan156**
_ Backend and frontend for login & register feature
_ Landing Page for SpartanStop
_ Setups for deploying to Heroku
_ Verification Steps:
       + go to https://peaceful-mountain-86940.herokuapp.com/
       + update the latest master and run the code
       + register for new user account
       + login with new user account
       + should be directed to landing page after logged in 
       + logout by clicking button at the top right corner

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-reply to this message or 7 years of bad luck
\ No newline at end of file
+// install dependencies
+$npm client-install
```
Comment (SandroSal) [inline README.md]: This will lead to a merge conflict.

File: front_end/src/actions/types.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const GET_ERRORS = "GET_ERRORS";
+export const CLEAR_ERRORS = "CLEAR_ERRORS";
+export const SET_CURRENT_USER = "SET_CURRENT_USER";
+export const TEST_DISPATCH = "TEST_DISPATCH";
```
Comment (DaleCS) [inline front_end/src/actions/types.js]: TEST_DISPATCH key is not used anywhere. Possible unneeded line?

Comment (DaleCS) [reply front_end/src/actions/types.js]: CLEAR_ERRORS is also not used anywhere. I assume it's going to be used in the future.

File: front_end/src/asset/userImage.svg [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+		<path style="fill:#010002;" d="M9.954,241.305h228.441c3.051,0,5.896-1.246,7.805-3.416c1.659-1.882,2.393-4.27,2.078-6.723
+			c-5.357-41.734-31.019-76.511-66.15-95.053c-14.849,14.849-35.348,24.046-57.953,24.046s-43.105-9.197-57.953-24.046
+			C31.09,154.65,5.423,189.432,0.071,231.166c-0.315,2.453,0.424,4.846,2.078,6.723C4.058,240.059,6.903,241.305,9.954,241.305z"/>
+		<path style="fill:#010002;" d="M72.699,127.09c1.333,1.398,2.725,2.73,4.166,4.019c12.586,11.259,29.137,18.166,47.309,18.166
+			s34.723-6.913,47.309-18.166c1.441-1.289,2.834-2.622,4.166-4.019c1.327-1.398,2.622-2.828,3.84-4.329
+			c9.861-12.211,15.8-27.717,15.8-44.6c0-39.216-31.906-71.116-71.116-71.116S53.059,38.95,53.059,78.16
+			c0,16.883,5.939,32.39,15.8,44.6C70.072,124.262,71.366,125.687,72.699,127.09z"/>
+	</g>
+</g>
+<g>
```
Comment (DaleCS) [inline front_end/src/asset/userImage.svg]: Unnecessary <g> tags?

**Comment by DaleCS** [comment]
These files are still using using the name 'miniSafeway':
/front_end/public/index.html
/front_end/src//components/layout/Footer.js
/front_end/src//components/layout/Landing.js

**Pull request merged**